src/GraphQL.Serialization.Newtonsoft/ExecutionErrorExtensions.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+
+namespace GraphQL
+{
+    internal static class ExecutionErrorExtensions
```

Comment:
What is the reason for moving this code from `ExecutionError`?

Reply:
Ok. I got it. Just make these properties public in original class and then ignore them in the created contract resolver.